grants_tagger/evaluate_model.py

```diff
@@ -17,20 +17,26 @@
 from grants_tagger.predict import predict_proba_ensemble_tfidf_svm_bert, load_model
 
 
-def predict(X_test, model_path, nb_labels, threshold):
+def predict(X_test, model_path, nb_labels=None, threshold=None, return_probabilities=False):
```

Comment:
perhaps `threshold` should be `thresholds`?

Reply:
tricky one as you can pass in both a value and a list, not sure what the right name in these cases